protobuf/plugin/fieldpath/helpers.go.go

```diff
@@ -14,14 +14,15 @@
    limitations under the License.
 */
 
-package plugin
+package fieldpath
```

Comment:
Does this package need to moved? Also looks like we picked up another go extension.

Reply:
Ok, this is a good move, no need for the extension.